### PR TITLE
feat(ui5-task-zipper): add standalone bundle to zip (#908)

### PR DESCRIPTION
**Pull request by the-docta**
Implements #908

documentation contains a "starting with version XXX" wording, which should be adjusted when releasing

### CODE REVIEW

**Comment by petermuessig** [comment]
See my comments in the issue #908

File: packages/ui5-task-zipper/README.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ Set this to `true` if you also want to generate the unzipped resources in the `d
 - includeDependencies: `true|false` or `String<Array>`
 Set this to `true` if you also want to include the dependencies (UI5 libraries) in the zip archive. Otherwise, it will only include the workspace files (controller, views, etc). In order to select only specific dependencies to be included in the final zip you just need to specify the list of dependencies (value of `ui5.yaml`: `metadata > name`).
 
+**NOTE**: sarting with release 3.0.5, `ui5-task-zipper` will also include the generated standalone app bundle (sap-ui-custom.* files), if it is run after the standard task `generateStandaloneAppBundle`.
```
Comment (petermuessig) [inline packages/ui5-task-zipper/README.md:38]: typo: "starting" not "sarting" 😄 

I would furthermore say: 

**NOTE**: Starting with release `3.0.5`, the `ui5-task-zipper` includes the generated workspace resources such as the self-contained bundles (`sap-ui-custom.*` files). To do so, it is important that the `ui5-task-zipper` is running as last task in the build.

File: packages/ui5-task-zipper/lib/zipper.js [hunk 2]
```diff
@@ -68,7 +68,7 @@ module.exports = async function ({ log, workspace, dependencies, options, taskUt
 	// get all application related resources
 	let allResources;
 	try {
-		const wsResources = await workspace.byGlob(`${prefixPath}/**`);
+		const wsResources = await workspace.byGlob(`/resources/**`);
```
Comment (petermuessig) [inline packages/ui5-task-zipper/lib/zipper.js:71]: 👍

**Review by petermuessig — CHANGES_REQUESTED** [review]
Typo in the `README.md` which should be fixed

The rest is great, especially that you enhanced the tests! BIG THX! 😄

File: packages/ui5-task-zipper/README.md (hunk 1 above)
Comment (the-docta) [reply packages/ui5-task-zipper/README.md]: typo fixed, docs updated as per your suggestion. Thx, Sergei

**Review by petermuessig — APPROVED** [review]
LGTM - thanks - will release now

File: packages/ui5-task-zipper/README.md (hunk 1 above)
Comment (petermuessig) [reply packages/ui5-task-zipper/README.md]: Great, thanks

**Comment by the-docta** [comment]
Cool, Thanks!

**Comment by petermuessig** [comment]
Release is in progress

**Pull request merged**